src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -223,18 +224,18 @@ This does not bypass the age restriction. It just accepts it automatically."</st
     <string name="revanced_enable_mini_player_previous_button_summary">Enables previous button in the miniplayer.</string>
     <string name="revanced_enable_swipe_to_dismiss_mini_player_title">Enable swipe to dismiss miniplayer</string>
     <string name="revanced_enable_swipe_to_dismiss_mini_player_summary">Enables swipe down to dismiss miniplayer.</string>
-    <string name="revanced_enable_zen_mode_title">Enable zen mode</string>
-    <string name="revanced_enable_zen_mode_summary">Changes the player background to light grey to reduce eye strain.</string>
-    <string name="revanced_enable_zen_mode_podcast_title">Enable zen mode in podcasts</string>
-    <string name="revanced_enable_zen_mode_podcast_summary">Zen mode is also applied to podcasts.</string>
-    <string name="revanced_hide_audio_video_switch_toggle_title">Hide audio video switch toggle</string>
-    <string name="revanced_hide_audio_video_switch_toggle_summary">Hides the audio video switch toggle in player.</string>
+    <string name="revanced_enable_zen_mode_title">Enable Zen mode</string>
+    <string name="revanced_enable_zen_mode_summary">Changes the player background color to light grey to reduce eye strain.</string>
+    <string name="revanced_enable_zen_mode_podcast_title">Enable Zen mode in podcasts</string>
+    <string name="revanced_enable_zen_mode_podcast_summary">Also enables Zen mode for podcasts.</string>
+    <string name="revanced_hide_audio_video_switch_toggle_title">Hide Audio / Video switch toggle</string>
+    <string name="revanced_hide_audio_video_switch_toggle_summary">Hides the Audio / Video switch toggle in the player.</string>
```

Comment:
"toggle" may be redundant here

Reply:
Or it could  been `Hide Audio/Video Switch Toggle` since it's a toggle rather than a button.

Reply:
@KobeW50, what's your opinion here?

Reply:
version 1:
title: `Hide Audio / Video toggle`
description:  `Hides the Audio / Video toggle in the player.`

version 2:
title: `Hide Audio / Video switch`
description: `Hides the Audio / Video switch in the player.`

"Switch" is synonymous with "toggle," and a toggle is a form of a button. 

A small downside to the second version is that "switch" is only used in RVX as a verb.

Reply:
I'll go with version 1 personally. @inotia00, any objections?

Reply:
`toggle` seems better